docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
 
 * Task name should be in a pair of quotation marks.
 
-* The floating task will be stored and shown to user according to the order of
-  the time they created.
+* Task name could be a single word or a phrase, white space are allowed.
```

Comment:
At the very beginning, I think we could just inform the user not to use quotes in the name of the task and multiple quotes will be forbidden in the command
And we could inform the user to use ‘’ instead at this time